src/services/anchor/index.ts

```diff
@@ -194,10 +197,20 @@ export async function sep24First(sessionParams: IAnchorSessionParams): Promise<I
   const { sep24Url } = tomlValues;
 
   // at this stage, assetCode should be defined, if the config is consistent.
-  const sep24Params = new URLSearchParams({
-    asset_code: sessionParams.tokenConfig.stellarAsset.code.string,
-    amount: sessionParams.offrampAmount,
-  });
+  let sep24Params;
+
+  if (sessionParams.tokenConfig.stellarAsset.code.string === 'ARS\0') {
+    sep24Params = new URLSearchParams({
+      asset_code: sessionParams.tokenConfig.stellarAsset.code.string.replace('\0', ''),
```

Comment:
Since we explicitly check for the ARS token here, I guess you can also just do
```suggestion
      asset_code: 'ARS',
```

Reply:
True, good one.